src/lwc/utilTemplateBuilder/utilTemplateBuilder.js

```diff
@@ -205,6 +258,17 @@ const lightningInputTypeByDataType = {
     'combobox': 'lightning-combobox'
 }
 
+
+/*******************************************************************************
+ * @description Retrieves the current namespace in an org
+ *
+ * @return {object} promise: Promise from the imperative apex call
+ * getNamespaceWrapper.
+ */
+const getNamespaceWrapper = async () => {
```

Comment:
Yeah, this call to Apex to get the namespace should not be re-introduced.  Happy to have a call to walk through the other method of inspecting an imported field or object if it would be helpful!